server/utils/formatter.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @description Determines the overal status from individual status
+ * @param {Array} automations Array of automations
+ *
+ * @returns {String} success/failure
+ */
+export function getOveralStatus(automations) {
+  if (Array.isArray(automations) && automations.length) {
+    if (automations.every(automation => automation.status === 'success')) {
+      return 'success';
```

Comment:
can we eliminate nested if statements by doing this:
```
return (Array.isArray(automations) && automations.length && automations.every(automation => automation.status === 'success'))? 'success' :  'failure';
```

Reply:
Done